buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/PokoDeserializerGenerator.kt

```diff
@@ -77,6 +77,11 @@ class PokoDeserializerGenerator(
         val funBuilder = FunSpec.builder(FROM_JSON)
             .addAnnotation(AnnotationSpec.builder(JvmStatic::class).build())
             .returns(ClassName.bestGuess(definition.name))
+        if (definition.isConstantClass())
+            funBuilder.addAnnotation(
+                AnnotationSpec.builder(SuppressWarnings::class)
+                    .addMember("%S", "UNUSED_PARAMETER").build()
+            )
```

Comment:
When the statement conditionned by an `if` is on a different line, let's always add braces.

Reply:
@xgouchet My first approach was that one but then I realised this will change the method signature `fromJson(jsonObject)` to `fromJson()` only for this specific property type. I would rather have this signature the same everywhere will be less intrusive and then add just suppress this warning because we know it is safe. What do you think ?

Reply:
Actually right now that you are also mentioning I will actually do it like that, was my first intention also :)